### PR TITLE
Backport mu-wpcom-plugin 2.0.9, jetpack 13.0-a.9 Changes

### PR DESCRIPTION
**Pull request by zinigor**
## Proposed changes:

Backports changes for mu-wpcom-plugin 2.0.9, jetpack 13.0-a.9

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

N/a
## Does this pull request change what data or activity we track or use?

No
## Testing instructions:

Proofread changes.

### CODE REVIEW

File: projects/plugins/mu-wpcom-plugin/mu-wpcom-plugin.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
  *
  * Plugin Name: WordPress.com Features
  * Description: Test plugin for the jetpack-mu-wpcom package
- * Version: 2.0.8
+ * Version: 2.0.9
```
Comment (CGastrell) [inline projects/plugins/mu-wpcom-plugin/mu-wpcom-plugin.php:6]: You mention this PR backports for mu-wpcom 2.0.10, is this 2.0.9 ok? Same on package.json below.

Comment (zinigor) [reply projects/plugins/mu-wpcom-plugin/mu-wpcom-plugin.php:6]: Good catch, this was probably a typo in the script that I ran locally. The version in the code is the one going to be used.

Comment (zinigor) [reply projects/plugins/mu-wpcom-plugin/mu-wpcom-plugin.php:6]: I have edited the PR description and header to avoid confusion, thanks!

**Review by CGastrell — APPROVED** [review]
LGTM!

**Pull request merged**